stable-diffusion/requirements.txt

```
@@ -0,0 +1,21 @@
+# --------- pytorch --------- #
+torch>=1.12.0
+torchvision>=0.11.0
+mosaicml[all]==0.12.0
```

```suggestion
mosaicml[all]>=0.12.0,<0.13
```
Can we get away with 12.1? trying to get all the examples using the latest version